src/libs/UserUtils.ts

```diff
@@ -128,11 +130,11 @@ function isDefaultAvatar(avatarURL?: string): boolean {
  * Provided a source URL, if source is a default avatar, return the associated SVG.
  * Otherwise, return the URL pointing to a user-uploaded avatar.
```

Comment:
Just thinking - should we update these function docs since we are updating the first param to be a "source" instead of always a "URL"? For example, for this function I feel like it would be a bit clearer to say something like:

```suggestion
 * Provided an avatar source, if source is a default avatar, return the associated SVG.
 * Otherwise, return the URL or SVG pointing to the user-uploaded avatar.
```